getting-started/authenticating-users/app.go

```diff
@@ -0,0 +1,139 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+    "fmt"
+    "log"
+    "net/http"
+    "io/ioutil"
+    "os"
+    "encoding/json"
+    "github.com/dgrijalva/jwt-go"
+)
+
+var AUDIENCE string
+var CERTIFICATES = make(map[string]string)
```

Comment:
Also, do audience and certificates need to be global if they only get used when they're already being returned by function calls?

Reply:
Those values need to be static. Once they are set once, they don't need to be set in the future.